server/tests/client_checks.py

```diff
@@ -13,6 +13,10 @@
 TIME_PERIOD_INCREMENT = 2
 DEFAULT_CHANNEL_NAME = "IOC1-1:ai:test"
 
+# 1 record and 1 alias
+BASE_ALIAS_COUNT = 1
+BASE_IOC_CHANNEL_COUNT = 2
```

Comment:
You can skip the comment if you do

```suggestion
BASE_CHANNEL_COUNT = 1
BASE_ALIAS_COUNT = 1
BASE_IOC_CHANNEL_COUNT = BASE_CHANNEL_COUNT + BASE_ALIAS_COUNT
```

🔧